tests/unit/classifier/svm/SVMLight_unittest.cc

```diff
@@ -0,0 +1,56 @@
+#include <shogun/classifier/svm/SVMLight.h>
+#include <shogun/features/DataGenerator.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/kernel/GaussianKernel.h>
+#include <gtest/gtest.h>
+
+using namespace shogun;
+
+#ifdef HAVE_LAPACK
+TEST(SVMLight,train)
+{
+	index_t num_samples = 100;
+	auto C = 1.0;
+	auto epsilon = 0.001;
+
+	CMath::init_random(5);
+	SGMatrix<float64_t> data =
+		CDataGenerator::generate_gaussians(num_samples, 2, 2);
+	CDenseFeatures<float64_t> features(data);
+
+	SGVector<index_t> train_idx(num_samples), test_idx(num_samples);
+	SGVector<float64_t> labels(num_samples);
+	for (index_t i = 0, j = 0; i < data.num_cols; ++i)
```

Comment:
so this logic is being copy pasted around the whole unit test codebase.
it'd be great that if you could move this into a utility method and somehow either be able to refer to it as a fixture in the unit test, or at least have a method that would just return the `features_train`, `features_test` and `ground_truth` instead of constructing it here in the code